build/BUILD

```diff
@@ -41,6 +41,10 @@ DOCKERIZED_BINARIES = {
         "base": ":busybox",
         "target": "//cmd/cloud-controller-manager:cloud-controller-manager",
     },
+    "gke-certificates-controller": {
```

Comment:
Yeah, I had a PR opened for building a Docker image for this before, but had some pushback, so I discarded it. If we're still committed to moving the controller out of the repo, this seems like a step in the wrong direction.